README.md

```diff
@@ -2,4 +2,64 @@
 raster-tile-query
 =================
 
-raster-tile-query allows you to query raster tiles and return pixel values at specific lng, lats
+:warning: **This project is a proof of concept and not intended for production use.** :warning:
+
+If you are looking to just get pixel values from an image, we recommend using [Node Mapnik](http://mapnik.org/documentation/node-mapnik/) and [sphericalmercator](https://github.com/mapbox/sphericalmercator) directly instead of this library. This is explained below.
+
+To query raster tiles at particular longitudes and latitudes, you'll need to know the X, Y, Z, and pixel size of your image tile to be able to interpolate which relative pixel to get information from. XYZ info is generally found in the request URL of your tile like this:
+
+```
+https://api.mapbox.com/v4/mapbox.streets/3/2/3.png?access_token={token}
```

Comment:
@mapsam for `mapbox.terrain-rgb` to work, `.pngraw` tiles **must** be requested -- I know that this example points to `mapbox.streets` but I fear users might simply swap out the mapid. Maybe we could point to that as an example instead as it is a more common use case for query?

Reply:
Great call - I had thought I put that URL in here actually. I've updated with the pngraw and a link to the Mapbox Terrain RGB blog post